### DISCUSSION TITLE
AIR iOS mobile Emulator

### DISCUSSION THREAD

**Discussion by gnutwist** [airsdk/Adobe-Runtime-Support#2507] [Q&A]
I hope this is ok... I searched online how to ask a question, and was informed to open an issue: I went about installing AIR for use as an iOS emulator based on some things I read on stackoverflow, and a youtube video by Cap Trong Nam. But it's beginning to look like I was mistaken. Is there something in AIR that functions as an iOS mobile emulator? If so, how is it implemented please?

**Comment by marchbold** [comment]
@gnutwist Best to ask this question in the discussion forum on the https://github.com/airsdk/Adobe-Runtime-Support repository. I've transferred this there for you. 

Regarding an iOS device simulator, you can use the iOS device simulator from Xcode to test and debug AIR applications. 

Or you can run in the AIR simulator, which won't have access to native functionality but will give you a quick way to test your application.

**Reply by gnutwist** [reply]
@marchbold thank you for directing me to the right place, and thank you for your reply. Adobe Air is mentioned on iGeeksBlog 8 best iOS emulators for PC in 2023. But I've been at this for a couple days, searching online for guidance... may I ask, how do I setup the AIR simulator for Visual Studio 2019 on Windows 10 to emulate iOS and achieve as you mentioned... a quick way to test a mobile app?

**Reply by ajwfrost** [reply]
This sounds like there is some confusion or misconception from that blog post! When you develop AIR apps, you have the option to run them in a local (desktop) simulator, but as @marchbold says, this doesn't give you access to native (iOS) functionality, it's just an AIR runtime that has some properties such as to mimic rotation or home-button presses etc. So it's not an "iOS emulator" by any stretch of the imagination! It's useful if you're creating AIR apps, but if you're just looking for some way to emulator iOS on Windows then it's not going to be any help I'm afraid..

thanks
